cmd/neofs-node/config.go

```diff
@@ -346,10 +356,17 @@ type shared struct {
 	metricsCollector *metrics.NodeMetrics
 }
 
+// dynamicConfiguration stores parameters of the
+// components that supports runtime reconfigurations
+type dynamicConfiguration struct {
```

Comment:
Do we expect to store shard parameters and morph here in future?

Reply:
yes, i expect to store them (and any component that supports cfg reload) here and it would be perfect if all of them will have the same interface `Reload() err`